src/hotspot/share/opto/loopnode.cpp

```diff
@@ -364,7 +364,7 @@ void PhaseIdealLoop::insert_loop_limit_check_predicate(ParsePredicateSuccessProj
 #endif
 }
 
-Node* PhaseIdealLoop::loop_exit_control(Node* x, IdealLoopTree* loop) {
+Node* PhaseIdealLoop::loop_exit_control(const Node* x, const IdealLoopTree* loop) const {
```

Comment:
There are quite some places where use `x` to denote a `loop_head`. Maybe you can also fix that with this patch.

Reply:
I would prefer a simple `head` for function named `.*loop.*`. Updated all relevant functions I touched.